api_app/models/domain/airlock_request.py

```diff
@@ -29,6 +29,11 @@ class AirlockRequestType(str, Enum):
     Export = strings.AIRLOCK_REQUEST_TYPE_EXPORT
 
 
+class AirlockActions(str, Enum):
+    Review = strings.AIRLOCK_ACTION_REVIEW
+    Cancel = strings.AIRLOCK_ACTION_CANCEL
+
```

Comment:
What about submit?

Reply:
it wasn't a requirement

Reply:
Okay, but do we have an explanation for why cancel and not submit? Maybe something is missing in the story?
cc: @tamirkamara

Reply:
it's not required by the UI, but I'm adding it for completeness